Parser/OpenRule.php

```diff
@@ -25,7 +25,7 @@ class OpenRule extends Rule
     /**
      * @param $source
      *
-     * @return Token[]
+     * @return Token[]|iterable<Token>
```

Comment:
Could be just `iterable<Token>` but I'm not sure if phpStorm really understands it to provide intelisense. I had mixed results at best with phpStorm vs. all newer typing capabilities of phpdocs